src/analytics/trackers.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * WordPress dependencies
+ */
+import { sendEventToHost } from '@wordpress/react-native-bridge';
+
+/**
+ * Internal dependencies
+ */
+import * as EVENTS from './events';
```

Comment:
I was thinking we could just define all the events here and import into the redux middleware module. The only tricky part is the redux action namespaces. It feels better to sort that out in the `events.js` file.

Reply:
I agree, I let too much YAGNI slip in with this structure 😄 
I do prefer to  keep the concerns separated so I opted for a new structure:
- All the tracking related to redux is now in `src/analytics/redux`. ( after reading your comment I realized the namespaced redux events would make no sense to any future potential event source so there was no need to have them in a generalized location )
- `src/analytics/redux/index` is only concerned with the middleware. It learns what should be tracked but delegates how it's tracked. i.e. no event property transformation or how the event gets sent to the host.
- `src/analytics/redux/tracked_events` defines what and how an action is tracked.

I find this new structure makes it clear that we (presently) only tracking redux events but that `src/analytics` , as a module, is open for future event sources. Any additional event tracking would be independent from the redux tracking since all the redux stuff is encapsulated in `src/analytics/redux`

I also opted to keep all the tracking logic separate from the middleware logic since updating an analytic event should not require knowing about the middleware flow. I should only need review `src/analytics/redux/tracked_events` to understand which events are tracked and how the analytic event payload is prepared. 
I also hope this separation would discourage folks from changing the middleware to handle a specific event case. Any specific event handling should happen in the `tracked_events` file.